### ISSUE TITLE
Disable the dask-ml.readthedoc version of the documentation ? (last version from 2017)

### ISSUE THREAD

**Issue by Carreau** [dask/dask-ml#588]
By googling I just came across https://dask-ml.readthedocs.io/en/stable/examples/hyperparameter-search.html which seem to be super old compare to https://ml.dask.org/hyper-parameter-search.html

In the best scenario the RTD one will be rebuilt with some rel='canonical' that point to the new version; 

You could prepend a warning to all the page as well, which is easier with sphinx, but at the minimum, I would just deactivate dsk-ml.readthedocs.org. 

Thanks.

**Comment by TomAugspurger** [comment]
Thanks for the report, sorry for the delay.

> In the best scenario the RTD one will be rebuilt with some rel='canonical' that point to the new version;

Agreed. I could never figure out how to configure RTD to do that.

> You could prepend a warning to all the page as well, which is easier with sphinx, but at the minimum

Hmm, we do that for `/en/latest` (https://dask-ml.readthedocs.io/en/latest/) but apparently not for `/en/stable`... Not sure why.

**Comment by Carreau** [comment]
> Agreed. I could never figure out how to configure RTD to do that.

Yeah, I think one way is to download all the HTML, patch it, zip-it, , and then "rebuild" the docs by just unpacking a ZIP :-)
